drivers/storage/portworx/deployment.go

```diff
@@ -1250,6 +1308,19 @@ func (t *template) getDesiredTelemetryImage(cluster *corev1.StorageCluster) stri
 	return ""
 }
 
+func (t *template) setOSImage(osImage string) {
+	logrus.Infof("Template OSImage set to %q", osImage)
+	t.osImage = osImage
+}
+
+func (t *template) isBottleRocketOS() bool {
+	if t.osImage == "" {
+		return false
+	}
+	lc := strings.ToLower(t.osImage)
+	return strings.HasPrefix(lc, "bottlerocket os")
```

Comment:
we can keep this more open by just checking for case insensitive bottlerocket. (in case they change the string slightly)